index.html

```diff
@@ -20,22 +21,26 @@
     </section>
     <section class = "folder-content">
       <form>
-        <label>Sort by Popularity</label>
+        <div class = "folder-title"></div>
+        <h4>Sort URLs</h4>
+        <label>By Popularity</label>
         <input type = "radio" id = "popularity" name = "sort" value = "popularity"/>
-        <label>Sort by Recency</label>
+        <label>By Recents</label>
```

Comment:
I don't love this language, but not sure what else to say ... I'll think about it. Maybe sort by most recent ?